src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  const threeDaysDiscount = 20;
+  const sevenDaysDiscount = 50;
```

Comment:
[checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 5

Reply:
got it